### ISSUE TITLE
What is the best approach to troubleshoot the PCA loss training?

### ISSUE THREAD

**Issue by hummuscience** [paninski-lab/lightning-pose#317]
I am currently training a model with 40+ keypoints (hand movement of macaques) and hanving troubles with the pca_singleview loss. All other types of losses and model combinations work fine and reach acceptable RMSEs withing 600-1000 epochs. The pca_singleview model has some reduced validation loss, but then stagnated at 40 RMSE and stays there for a while. 

Looking at the train_pca_singleweight_loss, all values are NaN/0. 

There seems to be something wrong with the PCA calculation. What would be the best way to go about troubleshooting this?

**Comment by hummuscience** [comment]
Single camera. We had a DLC model but were disappointed in how well its performing and adding more and more images was not helping it in the end
